tests/backup/backup_helper.go

```diff
@@ -1218,85 +1089,104 @@ func DeletePodWithLabelInNamespace(namespace string, label map[string]string) er
 }
 
 // backupSuccessCheck inspects backup task
-func backupSuccessCheck(backupName string, orgID string, retryDuration int, retryInterval int, ctx context.Context) (bool, error) {
-	backupDriver := Inst().Backup
+func backupSuccessCheck(backupName string, orgID string, retryDuration time.Duration, retryInterval time.Duration, ctx context.Context) error {
```

Comment:
why are we not returning status for succescheck's ?

Reply:
err being nil does the trick.